libs/service-portal/assets/src/screens/VehicleBulkMileageUpload/VehicleBulkMileageUpload.tsx

```diff
@@ -118,7 +131,11 @@ const VehicleBulkMileageUpload = () => {
           <AlertMessage
             type="warning"
             title={formatMessage(vehicleMessage.uploadFailed)}
-            message={data.vehicleBulkMileagePost.errorMessage}
+            message={
+              data?.vehicleBulkMileagePost?.errorCode === 429
```

Comment:
Maybe it would be better to set the error code in a variable with descriptive name

Reply:
That's true, I'll do that next time